.github/workflows/spec.yml

```diff
@@ -82,7 +77,13 @@ jobs:
       - name: Set up Clickhouse database schema
         run: bin/rails db:migrate:clickhouse
       - name: Run tests
-        run: bundle exec rake knapsack_pro:queue:rspec
+        run: |
+          if [[ -z "$KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC" ]]; then
+            echo "::warning::This is a community PR. We'll default to running the test using 'parallel_tests'."
+            bundle exec parallel_rspec --only-group "${KNAPSACK_PRO_CI_NODE_INDEX}" -n "${KNAPSACK_PRO_CI_NODE_TOTAL}" --exclude-pattern "spec/integration/.*_integration_spec.rb"
```

Comment:
`-n "${KNAPSACK_PRO_CI_NODE_TOTAL}"`
I'm thinking this should be the number of cores of the runner 🤔 
I haven't tried to GH actions but this should do it: `-n "${nproc}"`

Reply:
What's happening here is that we don't actually use `parallel_rspec` to run tests in parallel. When you combine `--only-group` with `-n`, we tell `parallel_rspec` to:

1. Split the tests into `KNAPSACK_PRO_CI_NODE_INDEX` (8) groups
2. Run only the `KNAPSACK_PRO_CI_NODE_INDEX` group

So we have the same behavior as Knapsack but without a proper test distribution based on timing.